.kuzzlerc.sample

```diff
@@ -0,0 +1,285 @@
+    //     using the "gitUrl" property
+    //   * from a local directory: the location of the package must be 
+    //     specified using the "path" property
+    "kuzzle-plugin-logger": {
+      "npmVersion": "2.0.4",
+      "activated": true
+    },
+    "kuzzle-plugin-auth-passport-local": {
+      "npmVersion": "2.0.4",
+      "activated": true
+    }
+  },
+  */
+
+  // The queues used in Kuzzle internal communication
+  /*
+  "queues": {
+    "remoteActionQueue": "remote-action-queue"
+  },
+  */
+
+  // The repositories are used internally by Kuzzle to store its data (users,
+  // permissions, configuration etc.)
+  /* 
+  "repositories": {
+    // [Common]
+    //   * cacheTTL: the time in the objects should be kepts in cache
+    // Decreasing this value will lower Redis memory and disk consumption at
+    // the cost of increasing the queries rate to the database
+    "common": {
+      "cacheTTL": 1440
+    }
+  },
+  */
+
+  // The security section contains the configuration for Kuzzle permissions
+  // mechanism
+  /*
+  "security": {
+    // [jwt]
+    // configuration for npm 
+    // [jsonwebtoken package](https://github.com/auth0/node-jsonwebtoken)
+    // 
+    //   * algorithm: hash/encryption method used to sign the token
+    //   * expiresIn: expressed in seconds or a rouchg/ms compatible string
+    //   * secret: a string or buffer containing either the secret for HMAC
+    //     algorithms, or the PEM encoded private key for RSA and ECDSA.
+    //     Please set this value to a private string
+    "jwt": {
+      "algorithm": "HS256",
+      "expiresIn": "1h",
+      "secret": "Change me"
+    },
+    // [default]
+    // The default role described here defines the permissions for the 
+    // anonymous user until an administrator user is created.
+    // By default, all users are granted all permissions until an administrator
+    // is defined.
+    "default": {
+      "role": {
+        "controllers": {
+          "*": {
+            "actions": {
+              "*": true
+            }
+          }
+        }
+      }
+    },
+    // [standard]
+    // Permissions used when creating an administrator user, either using the
+    // CLI or the Back Office.
+    // 
+    // By default, the admin user is granted all permissions.
+    // Anonymous and non-administrator users are restricted to the "serverInfo"
+    // action.
+    "standard": {
+      "profiles": {
+        "admin": {
+          "policies": [ {"roleId": "admin", "allowInternalIndex": true} ]
+        },
+        "default": {
+          "policies": [ {"roleId": "default"} ]
+        },
+        "anonymous": {
+          "policies": [ {"roleId": "anonymous"} ]
+        }
+      },
+      "roles": {
+        "admin": {
+          "controllers": {
+            "*": {
+              "actions": {
+                "*": true
+              }
+            }
+          }
+        },
+        "default": {
```

Comment:
not agree.
The "adminExists" action should only be available for admin users (role "admin" as permissions for all actions of all controllers, see line 120), and for the temporary permissions that are used before the admin is created (that is set ad line 89).

It's useless for "default" and "anonymous" roles.